app/assets/stylesheets/contests.scss

```diff
@@ -1,3 +1,83 @@
 // Place all the styles related to the contests controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+* {
+  -webkit-box-sizing: border-box;
+     -moz-box-sizing: border-box;
+          box-sizing: border-box;
+}
```

Comment:
`box-sizing: border-box;` is written in normalize.css and included in all pages.
So you don't need these styles.